WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingLaunchStoreViewModel.swift

```diff
@@ -136,9 +136,9 @@ private extension StoreOnboardingLaunchStoreViewModel {
 private extension StoreOnboardingLaunchStoreViewModel {
     enum Localization {
         static let upgradePlanToLaunchStore = NSLocalizedString(
-            "To launch your store, you need to upgrade to our plan. %1$@",
+            "To launch your store, you need to upgrade to your plan. %1$@",
```

Comment:
This is not a typo. I raised the same question to @annchichi and learnt that "our" stands for WordPress.com here. 😅 
Unfortunately, I couldn't find where I had the discussion. (Figma/Slack)
<img width="320" alt="Screenshot 2023-06-27 at 11 21 31 AM" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/9cf81677-f587-47f9-9e8e-ba22e7741b49">

Reply:
Wow that's a weird wording 😅 Reverted the change here: 7019941d5a4e9787f5e84aae17be06d9fee6763e.